lib/livebook/notebook/explore/distributed_portals_with_elixir.livemd

```diff
@@ -194,35 +194,11 @@ Date.from_iso8601("2020-02-30")
 
 Now, what happens if we want our code to behave differently depending
 if the date is valid or not? We can use `case` to pattern match on
-the different tuples. This is also a good opportunity to use Livebook's
-inputs to pass different values to our code. To render inputs, we need to
-install the [Kino](https://github.com/livebook-dev/kino) library:
+the different tuples:
 
 ```elixir
-Mix.install(
-  [
-    {:kino, github: "livebook-dev/kino"}
-  ],
-  consolidate_protocols: false
-)
-```
```

Comment:
This portal livebook now doesn't mention how to install kino anymore, which cause errors when it is used during the "Notebook connections" section

Reply:
@esambo it's been addressed in #953 and should be good on Livebook main :)

Reply:
Never mind, this has already been fixed in _main_ with #953